students/junghoon/user/views.py

```diff
@@ -36,3 +36,19 @@ def post(self, request):
         except KeyError:
             return JsonResponse({'message': 'KEY_ERROR'}, status=400)
 
+class LoginView(View):
+    def post(self, request):
+        try:
+            data     = json.loads(request.body)
+            email    = data['email']
+            password = data['password']
+
+            if User.objects.filter(email=email).exists():
+                user = User.objects.get(email=email)
+                if user.password == password:
+                    return JsonResponse({'message': 'SUCCESS'}, status=200)
+                return JsonResponse({"message": "INVALID_PASSWORD"}, status=401)
+            return JsonResponse({"message": "INVALID_USER"}, status=401)
```

Comment:
1. 비밀번호가 틀렸는지 이메일이 틀렸는지 모르게 하는 것 이 보안상 더 좋습니다.
2. if구문 사이에 공백을 추가하여 가독성을 높여 주세요.
3. 이메일 판단 후 비밀번호을 판단 하여 유저가 맞는지 확인하는 과정을 잘 구현 하셨습니다. 만약 지금같은 코드라면 User.objects.get(email=email)에서 발생할 수 있는 예외처리 코드를 추가해 주셔야 합니다.

```suggestion
            if User.objects.filter(email=email).exists():
                user = User.objects.get(email=email)
                
                if user.password == password:
                    return JsonResponse({'message': 'SUCCESS'}, status=200)
                
                return JsonResponse({"message": "INVALID_USER"}, status=401)
                
            return JsonResponse({"message": "INVALID_USER"}, status=401)
```

4. filter을 이용하면 아래와 같이 코드를 더 갈끔하게 구현 할 수 있을 것 같습니다.
```suggestion
            if not User.objects.filter(email=email, password=password).exists():
                return JsonResponse({"message": "INVALID_USER"}, status=401)
             
            return JsonResponse({'message': 'SUCCESS'}, status=200)
            
```

Reply:
1. 넵
2. 넵
3. dosenotexist를 추가하면 될까요?
4. 넵

Reply:
filter을 이용하는 것으로 수정 했으니  except User.DoesNotExist:를 추가하지 않아도 될 것 같습니다.